helium-lib/src/error.rs

```diff
@@ -37,6 +47,16 @@ pub enum Error {
     Decode(#[from] DecodeError),
     #[error("encode: {0}")]
     Encode(#[from] EncodeError),
+    #[error("Wallet is not configured")]
+    WalletUnconfigured,
+    #[error("error: {0}")]
+    Error(String),
```

Comment:
I'd like to understand the reason for this catch all error?

Reply:
I saw that there existed a EncodeError::other and a DecodeError::other that acted as a catch all so I mirroed the logic here. But appears I dont actually use it anywhere anymore so nuking it